src/main/java/com/example/demo/database/Product.java

```diff
@@ -0,0 +1,46 @@
+package com.example.demo.database;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "Dishes")
```

Comment:
why does class `Product` refer to a table `Dishes`. The table name should begin with lower case letter.